METRICS.md

```diff
@@ -147,6 +148,7 @@ Here is an example of Grafana dashboard importing.
 |         PotentialConsumeSpeed          |      worker       |                        This value means speed of potential consumption for congestion control.                         |
 |            UserProduceSpeed            |      worker       |                           This value means speed of user production for congestion control.                            |
 |           WorkerConsumeSpeed           |      worker       |                          This value means speed of worker consumption for congestion control.                          |
+|        isDecommissioningWorker         |      worker       |                              1 means worker decommissioning, 0 means not decommissioning.                              |
```

Comment:
The descripition of this metric is confusing that 0 not only means not decommissioning, but also mean decommissioned. Right?

Reply:
If decommissioned the worker will exit, seems  we cannot judge the status of a worker that has exited, but we can judge it through the Master's WorkerCount metrics.

Reply:
If this metric only represent that the worker is decommissioning, IMO, we could remove this metric. We could judge this via `DecommissionWorkers` of Master.

Reply:
It's ok for me.

Reply:
In a production environment, due to certain hardware or environmental reasons, our script will automatically decommission the node. We also need to alert based on isDecommissioning metrics, but we don’t want the alarm to go through graceful shutdown when upgrading the cluster.

Reply:
The starting point for mentioning this PR is that we want to distinguish between decommission and graceful shutdown.

Reply:
@leixm, where does the worker metric distingish between decommission and graceful shutdown? There is no shutdown metrics in worker.

Reply:
The time-consuming process of decommission is longer, I think we will pay more attention to its progress.

Reply:
ShutdownWorkerCount metrics includes decommission and graceful shutdown workers.